.changeset/grumpy-teachers-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
```

Comment:
There are no breaking changes in this PR right? In that case it should be `patch`

Reply:
It shouldn't brake anything 🙏 👍 Updated to a patch instead.